### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by anti-mony**
New Release
See CHANGELOG.md for changes and updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,21 +1,31 @@
-Change Log
-==========
+# Change Log
 
-v1.2.2 (2020/09/25)
--------------------
+## v2.0.0 (2020/09/25)
```
Comment (mgovoni-devel) [inline CHANGELOG.md]: Use today's date 2020/10/19

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
+
+Major Update
+
+- Added new frontend service, using NextJS(React)
+    - Improved UI using Material UI
+    - Updated outdated packages
+    - Reduced external dependencies
+    - Fixed search by tags 
+    - And added various small quality oif life improvements for users
```
Comment (mgovoni-devel) [inline CHANGELOG.md]: typo: oif

**Review by mgovoni-devel — CHANGES_REQUESTED** [review]

**Comment by anti-mony** [comment]
Fixed the date, 2020/10/19

File: CHANGELOG.md (hunk 1 above)
Comment (anti-mony) [reply CHANGELOG.md]: Yes, just fixed it

**Comment by anti-mony** [comment]
Fixed the typo oif -> of

**Review by mgovoni-devel — APPROVED** [review]

**Pull request merged**